aml-development/frontend
# feat(theme): new theme

AMLOS-762

**Description**
Review AMLOS-703 and choose an additional theme for the user to pick from the Edit Profile page.

**Acceptance Criteria**
New theme available and implemented

**How to Test**
Pull this branch
Review _Ocean Theme_ style guide here: https://wiki.udev.bia-boeing.com/display/AML/Themes
Ensure that _Ocean Theme_ is implemented properly
Also, since this involved refactoring the existing themes file, ensure that light and dark theme also are implemented properly
